config/data/menu.yml

```diff
@@ -1,50 +1,50 @@
 kits:
-  - caption
-  - card
-  - title
-  - button
-  - vertical_nav
-  - body
-  - layout
-  - image
-  - list
   - avatar
-  - line_graph
+  - badge
   - bar_graph
-  - icon
-  - table
+  - body
+  - button
+  - caption
+  - card
+  - checkbox
+  - contact
+  - currency
   - dashboard_value
-  - time
-  - pill
-  - user_badge
   - date
-  - title_detail
+  - date_range_inline
+  - date_year_stacked
+  - distribution_bar
+  - fixed_confirmation_toast
+  - hashtag
+  - home_address_street
+  - icon
   - icon_circle
   - icon_value
-  - star_rating
-  - progress_simple
-  - title_count
-  - user
+  - image
+  - label_pill
   - label_value
+  - layout
+  - line_graph
+  - list
+  - loading_inline
   - message
-  - timestamp
   - online_status
+  - person
+  - person_contact
+  - pill
+  - progress_simple
+  - section_separator
   - source
-  - label_pill
-  - phone
-  - owner_phone
-  - owner
-  - toggle
-  - stat_value
+  - star_rating
   - stat_change
-  - section_separator
-  - currency
-  - fixed_confirmation_toast
-  - loading_inline
-  - checkbox
-  - home_address_street
-  - badge
-  - date_year_stacked
-  - date_range_inline
-  - distribution_bar
-  - hashtag
+  - stat_value
+  - table
+  - time
+  - timestamp
+  - title
+  - title_count
+  - title_detail
+  - toggle
+  - user
+  - user_badge
+  - vertical_nav
```

Comment:
Never used this, but perhaps something we could add to enforce this https://www.npmjs.com/package/yaml-lint ?